pypreprocess/subject_data.py

```diff
@@ -461,6 +464,9 @@ def hardlink_output_files(self, final=False):
                 for sess in range(self.n_sessions):
                     filename = filenames[sess]
                     if not filename is None:
+                        # gzip before hard link
+                        if item == 'func':
```

Comment:
Anatomical data are handled above.
This test is just to avoid taking `realignment_parameters`